libbeat/idxmgmt/ilm/client_handler.go

```diff
@@ -179,6 +193,14 @@ func (h *ESClientHandler) CreateAlias(alias Alias) error {
 	// Note: actual aliases are accessible via the index
 	status, res, err := h.client.Request("PUT", "/"+firstIndex, "", nil, body)
 	if status == 400 {
+		hasAlias, err := h.HasAlias(alias.Name)
+		if err != nil {
+			return err
+		}
+		if !hasAlias {
+			return errf(ErrInvalidAlias,
+				"resource '%v' exists, but it is not an alias", alias.Name)
+		}
```

Comment:
the later error 'ErrInvalidAlias' with message is now returned by 'HasAlias' ;)

Reply:
Ok, removed the check here :)